multinode-demo/leader.sh

```diff
@@ -3,5 +3,10 @@
 # if RUST_LOG is unset, default to info
 export RUST_LOG=${RUST_LOG:-solana=info}
 
-sudo sysctl -w net.core.rmem_max=26214400
+if [[ "$(uname)" = "Linux" ]]; then
+  (
+    set -x
```

Comment:
why would you limit set -x to this subshell?

Reply:
Less stdout I guess, but you're right.  No point in limiting